addons/rating/__manifest__.py

```diff
@@ -22,10 +22,23 @@
     'assets': {
         'web.assets_backend': [
             'rating/static/src/scss/rating_rating_views.scss',
+            'rating/static/src/components/thread_needaction_preview/thread_needaction_preview.scss',
+            'rating/static/src/components/thread_preview/thread_preview.scss',
+            'rating/static/src/models/*.js',
         ],
         'web.assets_frontend': [
             'rating/static/src/scss/rating_templates.scss',
         ],
+        'web.assets_qweb': [
+            'rating/static/src/components/thread_needaction_preview/thread_needaction_preview.xml',
+            'rating/static/src/components/thread_preview/thread_preview.xml',
```

Comment:
Use glob patterns, see previous comment for explanations
```suggestion
            'rating/static/src/components/*/*.xml',
```